OrangeHRMTestFramework/Tests/OrangeHRM/EmployeeManagementTests.cs

```diff
@@ -30,6 +35,11 @@ public void SetUp()
         }
 
         [Test]
+        [AllureTag("Regression")]
```

Comment:
I suggest to improve the framework a little. You can create constants for AllureAtributes. Separate folders (AllureConstants) and classes for AllureTag, AllureSuite, AllureSubSuite.
It's just to avoid different categories with the same name but with typo in naming

Reply:
Fixed.